app/src/main/java/pl/gittobefit/WorkoutDisplay/viewmodel/InitiationTrainingDisplayLayoutViewModel.java

```diff
@@ -16,4 +27,68 @@
     public void setNumberOfClickedTraining(Integer i) {
         numberOfClickedTraining.setValue(i);
     }
+
+    public ArrayList<TrainingWithForm> getTrainingWithForms() {
+        return trainingWithForms;
+    }
+
+    public void setTrainingWithForms(ArrayList<TrainingWithForm> trainingWithForms) {
+        this.trainingWithForms = trainingWithForms;
+    }
+
+    public void addTrainingWithForm(TrainingWithForm trainingWithForm)
+    {
+        trainingWithForms.add(trainingWithForm);
+    }
+
+    public void setStatesExerciseLists(int count)
+    {
+        states.clear();
+        for (int i = 0; i < count; i++)
+        {
+            states.add(false);
+        }
```

Comment:
states = new ArrayList(count ) ale to chlopakow od javy trzeba było by sie dopytac